src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay/configuration/customize-privacy-settings.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "(Optional) Configure privacy settings"
+metaDescription: "Customizing Privacy Settings for user interactions to debug faster and improve page performance."
+freshnessValidatedDate: 2024-12-19
+---
+
+User privacy is a critical consideration when recording session replays. Although default settings are designed to protect sensitive information, you may need to adjust privacy configurations to meet specific needs. This section explains how to mask, block, or ignore certain data to comply with privacy standards while maintaining the functionality of session replays.
+
+By default, all text and inputs are replaced with asterisks. Here's an example of how a replay looks with default privacy:
+
+    <img
+      title="Example of default privacy settings"
+      alt="Example of default privacy settings"
+      src="/images/browser_screenshot-full_text-masking-example.webp"
+    />
+
+  If this setting is appropriate, move onto the next step.
+
+  If you need to customize what content or user input is captured, you have options to mask, block, or ignore specific classes and attributes of content or user input. Make sure to check with your privacy team before changing the privacy settings.
+
+## Mask sensitive text [#mask-text]
+
+  You can mask sensitive text, which means the text will be replaced with asterisks (\*). For example, you can mask a user's account ID that shows in a URL.
+
+  To specify which input should be masked, you have a few options:
+
+  * On the <DNT>**Application settings**</DNT> page, add your own CSS selectors or check the masking boxes. Note that using mask selectors will **not** mask user input, so if you need to hide user input, we recommend using block selectors.
+
+      <img
```

Comment:
Is this image necessary?

Reply:
Let me check this with the PM and get back to you.